js/widgets/site-search.js

```diff
@@ -1,34 +1,36 @@
 /**
  * Main site search
  */
+ const config = require('config');
+
 define([
     'jquery',
     'dataset/typeahead',
     'reuse/typeahead',
     'organization/typeahead',
-    // 'user/typeahead',
+    'territory/typeahead',
     'typeahead'
-], function($, datasets, reuses, organizations) {
+], function($, datasets, reuses, organizations, territories) {
     'use strict';
 
-    var SEARCH_FOCUS_CLASS = 'col-sm-7 col-lg-8',
-        SEARCH_UNFOCUS_CLASS = 'col-sm-2 col-lg-3',
-        options = {
-            highlight: true,
-            classNames: {
-                menu: 'tt-dropdown-menu'
-            }
-        };
+    const SEARCH_FOCUS_CLASS = 'col-sm-7 col-lg-8';
+    const SEARCH_UNFOCUS_CLASS = 'col-sm-2 col-lg-3';
+    const options = {
+        highlight: true,
+        classNames: {
+            menu: 'tt-dropdown-menu'
+        }
+    };
 
     // Expandable main search bar
     $('#main-search')
-        .focusin(function() {
+        .focusin(() => {
```

Comment:
Can't do this in this case: jQuery set the this/scope to the select object. `$(this)` is incompatible with arrow functions.